src/components/Divider/divider.module.css

```diff
@@ -0,0 +1,17 @@
+:root {
+  --denhaag-divider-border-color: var(--denhaag-grey-2);
+  --denhaag-divider-border-width: 1px;
+  --denhaag-divider-margin: 16px 16px;
+}
+
+.denhaag-divider {
+  height: var(--denhaag-divider-border-width);
+  background: var(--denhaag-divider-border-color);
+  margin-block: var(--denhaag-divider-margin);
+}
+
+.denhaag-divider-vertical {
```

Comment:
Following the BEM syntax should be `denhaag-divider--vertical`, with an additional dash.